numba/tests/test_builtins.py

```diff
@@ -505,6 +508,51 @@ def test_hex_npm(self):
         with self.assertTypingError():
             self.test_hex(flags=no_pyobj_flags)
 
+    def test_int_str(self, flags=nrt_no_pyobj_flags):
+        pyfunc = str_usecase
+
+        small_inputs = [
+            1234,
+            1,
+            0,
+        ]
+
+        large_inputs = [
+            123456789,
+            2222222,
+            ~0x0
+        ]
+
+        args = [*small_inputs, *large_inputs]
+
+        typs = [
+            types.int8,
+            types.int16,
+            types.int32,
+            types.int64,
+            types.uint,
+            types.uint8,
+            types.uint16,
+            types.uint32,
+            types.uint64,
+            types.uintp,
+            types.ushort,
+            types.ulong,
+            types.ulonglong,
+        ]
+
+        for typ in typs:
+            cr = compile_isolated(pyfunc, (typ,), flags=flags)
+            cfunc = cr.entry_point
+            for v in args:
+                if v in large_inputs and typ.bitwidth < 32:
+                    continue
```

Comment:
Can probably remove this? If it overflows it still ought to work?

Reply:
Yes, you're right.